### PR TITLE
PROD-125 update-burntx-code-to-versioned-style

### PR DESCRIPTION
**Pull request by danijel-radomiljac**

### CODE REVIEW

File: app/utils/solana.ts [hunk 1]
```diff
@@ -37,26 +38,27 @@ export const genBonkBurnTx = async (
     DECIMALS,
   );
 
-  tx.recentBlockhash = blockhash;
-  tx.feePayer = burnFromPublic;
-
-  const estimateFee = await getRecentPrioritizationFees(tx);
```
Comment (marvinmarnold) [inline app/utils/solana.ts:43]: Why do you want to remove this?

Comment (danijel-radomiljac) [reply app/utils/solana.ts:43]: Because i cannot send versionedTx to that fn and after that push that instruction.

Comment (danijel-radomiljac) [reply app/utils/solana.ts:43]: I added fixed fee

**Review by marvinmarnold — COMMENTED** [review]
I'd rather keep optimal fee calculation, than use versioned txs. Let's close this PR for now.

**Pull request closed without merging**